client/client/app/components/ngbBlastSearchPanel/index.js

```diff
@@ -0,0 +1,17 @@
+import angular from 'angular';
+
+// Import Style
+import './ngbBlastSearchPanel.scss';
+
+// Import internal modules
+import ngbBlastSearchPanel from './ngbBlastSearchPanel.component';
+import controller from './ngbBlastSearchPanel.controller';
+
+// Import components
+import ngbBlastWholeGenomeView from './ngbBlastWholeGenomeView';
+
+// Import external modules
+export default angular
+    .module('ngbBlastSearchPanel', [ngbBlastWholeGenomeView])
+    .component('ngbBlastSearchPanel', ngbBlastSearchPanel)
+    .controller(controller.UID, controller).name;
```

Comment:
@ZMaratovna please don't forget about *new line* symbol at the end of file (here and everywhere)